build_projects/update-dependencies/Program.cs

```diff
@@ -21,75 +21,57 @@ public static void Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
+            bool onlyUpdate = args.Length > 0 && string.Equals("--Update", args[0], StringComparison.OrdinalIgnoreCase);
```

Comment:
[nit] Seems like a weird way to check for this arg, but I see it was just moved from below.